### DISCUSSION TITLE
What do you think we will use HXPhotoPicker

### DISCUSSION THREAD

**Discussion by baronha** [NitrogenZLab/react-native-multiple-image-picker#120] [Polls]
I just took a look around Github and looked back at our library. It is inherited from TLPhotoPicker - A great library written by @tilltue. But since January of last year (2022), there has been no further update. I just came across [HXPhotoPicker](https://github.com/SilenceLove/HXPhotoPicker) and was amazed at how useful it can be in many of our projects (Of course, we can't use all of them). Can I survey your opinion on whether you should switch to using HXPhotoPicker on iOS?

- Documentation: https://github.com/SilenceLove/HXPhotoPicker/blob/master/Documentation/README_EN.md
- Video Demo: http://oss-cn-hangzhou.aliyuncs.com/tsnrhapp/shop/videos/83862ab94facfd8979eb6148094908b2.mp4

**Comment by Splicer97** [comment]
Yeah, it's cool idea

**Reply by baronha** [reply]
Sound good. but I'm lazy =))

**Comment by baronha** [comment]
I received consent from @SilenceLove - He is an amazing person with extraordinary skills .
<img width="1092" alt="Screenshot 2023-10-12 at 18 42 57" src="https://github.com/baronha/react-native-multiple-image-picker/assets/23580920/2fdf53b5-ef24-4d2b-b0b4-4bd34237b41c">

**Comment by baronha** [comment]
Almost completed: https://github.com/baronha/react-native-multiple-image-picker/pull/180
